NebulaModel/Packets/Players/PlayerColorChanged.cs

```diff
@@ -0,0 +1,17 @@
+﻿using NebulaModel.DataStructures;
+
+namespace NebulaModel.Packets.Players
+{
+	public class PlayerColorChanged
+	{
+		public ushort PlayerId { get; set; }
+		public Float4 Color { get; set; }
```

Comment:
We should probably use a Float3 for the color since we don't need to send the Alpha channel. It will always be 1.